authors/apps/articles/models.py

```diff
@@ -1,9 +1,67 @@
 from django.db import models
 from django.utils.text import slugify
+from django.db.models import Sum
+from django.utils.translation import ugettext_lazy as _
+from django.contrib.contenttypes.models import ContentType
+from django.contrib.contenttypes.fields import GenericRelation, GenericForeignKey
 
 from authors.apps.authentication.models import User
 
 
+class LikeDislikeManager(models.Manager):
+    """
+    Manager class for like and dislike
+    """
+    use_for_related_fields = True
+
+    def likes(self):
+        """
+        get all the likes for an object
+        """
+        return self.get_queryset().filter(vote__gt=0)
+
+    def dislikes(self):
+        """
+        get all the dislikes of an object
+        """
+        return self.get_queryset().filter(vote__lt=0)
+
+    def sum_rating(self):
```

Comment:
I think the sum of likes and dislikes is important... it add trust and more fun to the game and the idea of articles

Reply:
@charitymarani, in the API, one has to get the total number of likes/dislikes, then in the frontend, it will just be a matter of displaying the total number, as @eduhmik explain. Maybe I should rename the method appropriately